src/store.ts

```diff
@@ -178,6 +178,20 @@ export class Store {
     }
   }
 
+  invalidate(
+    rootKey: string,
+    relationKey: string | string[],
```

Comment:
I think we don't really need multiple field keys here tbh, especially since field key + the same args won't make much sense in most cases.

If we use resolve wouldn't we want to implement resolveAll to eventually provide a search for all entities?

We'd also need to then support removing entities here (by removing their typename field possibly?)